components/FormComponents/GitHubUserNameInput.tsx

```diff
@@ -8,12 +8,13 @@ const GitHubUserNameInput: React.FC<FormState> = ({
   setFormData,
 }) => {
   return (
-    <div className="flex flex-col mt-10 max-w-2xl">
-      <label htmlFor="name" className="mb-2 md:text-lg lg:text-xl">
+    <div className="flex flex-col mt-10 max-w-2xl" role="Github Username">
+      <label htmlFor="username" className="mb-2 md:text-lg lg:text-xl">
         GitHub Username:
+        <span id="desc-username" className="hidden">Enter your github username</span>
       </label>
```

Comment:
I would like to know why there is a span tag inside a label tag for the same text.

Reply:
Actually the span tag acts as information for screen readers